### PR TITLE
[HOLD] Universal metrics export preview

### LINKED ISSUE #3879: Public preview - Universal Metrics Export
### New Feature Description

New mechanism to export metrics with the Astro UI

### Required Reviewers

chompi88

### Links to Internal Info or Resources

https://www.notion.so/astronomerio/Universal-Metrics-Export-8ccb7e9e1e144b7e91325e9a44f3a1c3?pvs=4

### Release Date

7/8 week

### Additional Notes

_No response_

### PR DESCRIPTION
**Pull request by lzdanski**
Resolves #3879

### CODE REVIEW

File: astro/universal-metrics-export.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+import TabItem from '@theme/TabItem';
+import TeamBadge from '@site/src/components/TeamBadge';
+import HostedBadge from '@site/src/components/HostedBadge';
+
+<TeamBadge/>
+
+:::publicpreview
+:::
+
+You can export comprehensive metrics about your Astro Deployments from Astro directly to any third-party monitoring and alerting systems. The universal metrics exporter uses the[OpenTelemetry (OTel)](https://opentelemetry.io/docs/) protocol to ensure that measurements about your Astro Deployments can be exported and used across any OTel compatible tool.
```
Comment (lzdanski) [inline astro/universal-metrics-export.md:17]:
```suggestion
You can export comprehensive metrics about your Astro Deployments from Astro directly to any third-party monitoring and alerting systems. The universal metrics exporter uses the [Prometheus]() format using the remote-write capability to ensure that measurements about your Astro Deployments to your preferred monitoring toolsl.
```

File: astro/universal-metrics-export.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+This functionality allows you to move operational data about your Astro Deployments to your prefferred monitoring tools, like Prometheus.
```
Comment (lzdanski) [inline astro/universal-metrics-export.md:19]:
```suggestion
```

File: astro/universal-metrics-export.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+import TabItem from '@theme/TabItem';
+import TeamBadge from '@site/src/components/TeamBadge';
+import HostedBadge from '@site/src/components/HostedBadge';
+
+<TeamBadge/>
+
+:::publicpreview
+:::
+
+You can export comprehensive metrics about your Astro Deployments from Astro directly to any third-party monitoring and alerting systems. The universal metrics exporter uses the [Prometheus data model](https://prometheus.io/docs/concepts/data_model/) format using the remote-write capability to ensure that measurements about your Astro Deployments to your preferred monitoring tools.
```
Comment (chomipi88) [inline astro/universal-metrics-export.md:17]:
```suggestion
You can export comprehensive metrics about your Astro Deployments from Astro directly to any third-party monitoring and alerting systems. The universal metrics exporter uses the [Prometheus data model](https://prometheus.io/docs/concepts/data_model/) format using the remote-write capability to export metrics about your Astro Deployments to your preferred monitoring tools.
```

File: astro/universal-metrics-export.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+
+## Metric types
+
+- Airflow application level metrics: Metrics defined by Airflow, or custom metrics you write, about the health, success, and performance of data pipelines orchestrated and executed by Airflow. You can see the specific list of [application metrics](https://github.com/astronomer/ap-vendor/blob/main/statsd-exporter/include/mappings-gen2.yml) and read their corresponding descriptions in the [Airflow metrics descriptions](https://airflow.apache.org/docs/apache-airflow/stable/administration-and-deployment/logging-monitoring/metrics.html#metric-descriptions).
```
Comment (chomipi88) [inline astro/universal-metrics-export.md:21]:
```suggestion
- Airflow application level metrics: Metrics defined by Airflow, about the health, success, and performance of data pipelines orchestrated and executed by Airflow. You can see the specific list of [application metrics](https://github.com/astronomer/ap-vendor/blob/main/statsd-exporter/include/mappings-gen2.yml) and read their corresponding descriptions in the [Airflow metrics descriptions](https://airflow.apache.org/docs/apache-airflow/stable/administration-and-deployment/logging-monitoring/metrics.html#metric-descriptions).
```

Comment (chomipi88) [reply astro/universal-metrics-export.md:21]: don't have custom metric support yet

File: astro/universal-metrics-export.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+
+- Infrastructure level metrics: Metrics about running pods for the different Airflow components. These indicate of the use, health, and performance of the pods based on your workload and pipelines.
```
Comment (chomipi88) [inline astro/universal-metrics-export.md:23]:
```suggestion
- Infrastructure level metrics: Metrics about running pods for the different Airflow components. These indicate the usage, health, and performance of the pods based on your workload and pipelines.
```

File: astro/universal-metrics-export.md [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+  - `container`
+  - `namespace`
+  - `pod`
+  - `deploymentId`
+  - `organizationId`
+  - `workspaceId`
+
+## Prerequisites
+
+- Bearer token or license key of your target data observability server
```
Comment (chomipi88) [inline astro/universal-metrics-export.md:46]:
```suggestion
- Supported Auth: Bearer token or license key, username and password, or custom HTTP header(s) of your target data observability server _(Optional)_
```

File: astro/universal-metrics-export.md [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+
+## Metric types
+
+- Airflow application level metrics: Metrics defined by Airflow, or custom metrics you write, about the health, success, and performance of data pipelines orchestrated and executed by Airflow. You can see the specific list of [application metrics](https://github.com/astronomer/ap-vendor/blob/main/statsd-exporter/include/mappings-gen2.yml) and read their corresponding descriptions in the [Airflow metrics descriptions](https://airflow.apache.org/docs/apache-airflow/stable/administration-and-deployment/logging-monitoring/metrics.html#metric-descriptions).
+
+- Infrastructure level metrics: Metrics about running pods for the different Airflow components. These indicate of the use, health, and performance of the pods based on your workload and pipelines.
+
+  | Name                                        | Description           |
+  | ------------------------------------------- | --------------------- |
+  | `container_cpu_usage_seconds_total`         | CPU use               |
```
Comment (chomipi88) [inline astro/universal-metrics-export.md:27]:
```suggestion
  | `container_cpu_usage_seconds_total`         | CPU usage               |
```

File: astro/universal-metrics-export.md [hunk 8]
```diff
@@ -0,0 +1,108 @@
...
+  | `container_memory_working_set_bytes`        | Memory use            |
```
Comment (chomipi88) [inline astro/universal-metrics-export.md:28]:
```suggestion
  | `container_memory_working_set_bytes`        | Memory usage            |
```

File: astro/universal-metrics-export.md [hunk 9]
```diff
@@ -0,0 +1,108 @@
...
+  | `kubelet_stats_ephemeral_storage_pod_usage` | Ephemeral storage use |
```
Comment (chomipi88) [inline astro/universal-metrics-export.md:29]:
```suggestion
  | `kubelet_stats_ephemeral_storage_pod_usage` | Ephemeral storage usage |
```

**Review by chomipi88 — COMMENTED** [review]
see comments

**Review by paolaperaza — APPROVED** [review]

**Pull request merged**